doc/manual/manual.md

```diff
@@ -3102,9 +3107,46 @@ s_8_16  <- mir_find_adt m "example::S" [mir_u8,  mir_u16];
 s_32_64 <- mir_find_adt m "example::S" [mir_u32, mir_u64];
 ~~~~
 
-The `mir_adt` command (for constructing a struct type) and `mir_struct_value`
-(for constructing a struct value) commands in turn take a `MIRAdt` as an
-argument.
+The `mir_adt` command (for constructing a struct type), `mir_struct_value` (for
+constructing a struct value), and `mir_enum_value` (for constructing an enum
+value) commands in turn take a `MIRAdt` as an argument.
+
+In addition to taking a `MIRAdt` as an argument, `mir_enum_variant` also takes
```

Comment:
```suggestion
In addition to taking a `MIRAdt` as an argument, `mir_enum_value` also takes
```
Typo?

Reply:
Yes indeed! Thanks for spotting this. I've pushed a fix.